packages/ses/NEWS.md

```diff
@@ -13,8 +13,9 @@ User-visible changes in SES:
   This overcomes the last *known* obstacle toward object capability containment.
 - Lockdown will now read options from the environment as defined by the Node.js
   `process.env` parameter space.
-- *BREAKING CHANGE*: It may no longer be safe to call `lockdown` more than once
-  per realm (VM context or iframe).
+- *BREAKING CHANGE*: Lockdown may no longer be called more than once.
+  Lockdown no longer returns a boolean to indicate whether it was effective
+  (true) or redundant (false).
```

Comment:
Should this mention an error is thrown if trying to lockdown twice? Seems weird to be explicit about former behavior, but not about new one.

Reply:
Explicated this. Thanks.

Reply:
I assume we should update the PR description too?